thingif/src/main/java/com/kii/thingif/clause/trigger/EqualsClauseInTrigger.java

```diff
@@ -13,6 +13,8 @@
     private @NonNull Object value;
     private @NonNull String alias;
 
+    private volatile int hashCode; // cached hashcode for performance
```

Comment:
> Since it is not part of logic, better to mention it.

I understand it.

> volatile is needed for multithread safe.

Do we need to make`hashCode` thread safe?
If `hashCode` is not thread safe, I think big problem does not happen.

If using `volatile` is good manager for `hashCode`, I agree with it.

How do you think.